cmd/commands/service/command.go

```diff
@@ -69,10 +78,20 @@ func NewCommand() *cli.Command {
 		Flags: []cli.Flag{
 			identityFlag, identityPassphraseFlag,
 			openvpnProtocolFlag, openvpnPortFlag,
+			agreedTermsConditionsFlag,
 		},
 		Action: func(ctx *cli.Context) error {
+			if !ctx.Bool(agreedTermsConditionsFlag.Name) {
+				fmt.Println(metadata.VersionAsSummary(metadata.LicenseCopyright(
+					"run program with '--license.warranty' option",
+					"run program with '--license.conditions' option",
+				)))
+				fmt.Println()
+				return fmt.Errorf("If you agree with these Terms & Conditions, run program again with '--agreed-terms-and-conditions' flag")
```

Comment:
Is it possible to do `fmt.Println()` and `os.Exit(2)`

Reply:
added `os.Exit(2)`